WeakAurasTemplates/TriggerTemplates.lua

```diff
@@ -483,6 +487,20 @@ local function subTypesFor(item, regionType)
       createTriggers = function(triggers, item)
         createAbilityTrigger(triggers, 1, item, "showOnCooldown");
       end,
+      createConditions = function(conditions, item, regionType)
+        isNotUsableBlue(conditions, 1, regionType)
+      end,
+    });
+    tinsert(types, {
+      icon = icon.cd,
+      title = L["Basic Show On Ready"],
+      description = L["Only shows the aura when the ability is ready to use."],
+      createTriggers = function(triggers, item)
+        createAbilityTrigger(triggers, 1, item, "showOnReady");
```

Comment:
showOnReady is wrong for abilities with multiple stacks.

Reply:
triggers[position].trigger.use_track = true
    triggers[position].trigger.track = "cooldown"
make it work with multiple stacks (i tested it with power word: radiance)

Reply:
On the other hand, that's how the other templates work, so maybe we shouldn't